### PR TITLE
Kbc 3081 async ws configuration create

### PR DESCRIPTION
**Pull request by romanbracinik**
Jira: KBC-3081
MainPr: https://github.com/keboola/connection/pull/4140
Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -7487,13 +7487,14 @@ Creates a new configuration row from an existing row version in [Development Bra
 
 ## Create or List Configuration Workspaces [/v2/storage/components/{component_id}/configs/{config_id}/workspaces]
 
-### Create Configuration Workspaces [POST /v2/storage/branch/{branch_id}/components/{component_id}/configs/{config_id}/workspaces]
+### Create Configuration Workspaces [POST /v2/storage/branch/{branch_id}/components/{component_id}/configs/{config_id}/workspaces?async={async}]
```
Comment (jirkasemmler) [inline apiary.apib:7490]: chtelo by to poznamku o tom, ze async=false je depreacted

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
jen to apiary a jinak cajk

File: apiary.apib (hunk 1 above)
Comment (tomasfejfar) [reply apiary.apib:7490]: Idealně u obou (i u standardní ws create), pokud není...

Comment (romanbracinik) [reply apiary.apib:7490]: Pridal som to a este som si vsimol, ze som nemal async=true response u toho configuration endpointu tak som to tam pridal. Snad to bude takto cajk.

File: apiary.apib [hunk 2]
```diff
@@ -4222,7 +4222,7 @@ It means that endpoint `GET /v2/storage/workspaces` is the same as `GET /v2/stor
 Creates a new workspace and return its credentials.
 
 + Parameters
-    + async (optional, boolean) - Create workspace in a background job.
+    + async (optional, boolean) - Create workspace in a background job. Calling the parameter as `async=false` is depricated.
```
Comment (jirkasemmler) [inline apiary.apib:4225]:
```suggestion
    + async (optional, boolean) - Create workspace in a background job. Calling the parameter as `async=false` is deprecated.
```

File: apiary.apib [hunk 3]
```diff
@@ -7487,13 +7487,14 @@ Creates a new configuration row from an existing row version in [Development Bra
...
 
-### Create Configuration Workspaces [POST /v2/storage/branch/{branch_id}/components/{component_id}/configs/{config_id}/workspaces]
+### Create Configuration Workspaces [POST /v2/storage/branch/{branch_id}/components/{component_id}/configs/{config_id}/workspaces?async={async}]
 Creates a new workspace for an existing configuration in [Development Branch](#reference/development-branches).
 
 + Parameters
     + branch_id (required) - Id of the development branch
     + component_id (required) - Id of the component - e.g. `keboola.ex-db-mysql`
     + config_id (required) - Id of the configuration
+    + async (optional, boolean) - Create workspace in a background job. Calling the parameter as `async=false` is depricated.
```
Comment (jirkasemmler) [inline apiary.apib:7497]:
```suggestion
    + async (optional, boolean) - Create workspace in a background job. Calling the parameter as `async=false` is deprecated.
```

**Review by jirkasemmler — APPROVED** [review]
cajk

**Pull request merged**